doc/REQUIREMENTS.md

```diff
@@ -457,6 +460,22 @@ report on is system load, uptime, wait times, projects that were run, projects t
 
 TODO: To be developed.
 
+### "NAIC Orchestrator" user stories
+
+*Story (WP8.x.x): User logs in to the NAIC Orchestrator and creates a VM. Once the VM is provisioned, sonar 
+package is installed in the VM. Orchestrator will also generate a parameter file in VM which contains the necessary 
+configuration (such as endpoints, vm ID, etc.) for sonar to work. This is done automatically without any user intervention.
```

Comment:
This first one is incomplete I think.  The full story is at least that when the VM is eventually run, data are collected and exfiltrated to an appropriate server for later analysis.